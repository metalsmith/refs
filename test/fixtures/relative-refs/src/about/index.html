---
refs:
  home: file:../index.html
  deeplyNested: ../some/deeply/nested/file.html
---