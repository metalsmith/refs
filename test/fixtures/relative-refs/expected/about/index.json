{
  "refs": {
    "home": {
      "id": "index.html"
    },
    "deeplyNested": {
      "id": "some/deeply/nested/file.html"
    }
  },
  "id": "about/index.html"
}